soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
Its just this stuff isn't going to merge with the other changes without alot of cajoling. I would really like use to just work on one branch for a while for these pathplanning changes.  We're still basically rewriting everything with every merge.